### PR TITLE
Update changelog/package.json for 4.0.0 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
 # 3.8.1 (2020-05-06)
 
 - Fixed the list component to return to the first page after applying a filter or sorting a list. [#89](https://github.com/blackbaud/skyux-list-builder/pull/89)
 - Fixed the list toolbar item component to honor item indexing. [#88](https://github.com/blackbaud/skyux-list-builder/pull/88)
 
+# 4.0.0-rc.2 (2020-04-17)
+
+### Bug fixes
+
+- Added `SkyListSecondaryActionsComponent` and `SkyListToolbarComponent` to the exports API.
```
Comment (Blackbaud-DeniseCisneros) [inline CHANGELOG.md:16]: Do we need PR links for the stuff that was added?

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md:16]: @Blackbaud-DeniseCisneros Good question. Those particular entries do not have pull requests associated with them.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-rc-merge-master-final](https://host.nxt.blackbaud.com/skyux-list-builder?_sp.spa=pr-rc-merge-master-final)

File: CHANGELOG.md (hunk 1 above)
Comment (Blackbaud-DeniseCisneros) [reply CHANGELOG.md:16]: No problem. Looks good then :)

**Pull request merged**